sql/derived_tables/po_lines_locations.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS local.nancybolduc_po_lines_locations;
+--Create a local table for Purchase Order Line Locations including location quantity and type.
+CREATE TABLE local.nancybolduc_po_lines_locations AS
```

Comment:
Note the table name `local.nancybolduc_po_lines_locations` which probably should be `local.po_lines_locations`.  I will fix this after merging.